ClassesHomework/Cat.cs

```diff
@@ -7,9 +7,20 @@
 namespace ClassesHomework
 {
     // Write your Cat class here
-    //class Cat
-    //{
-        
-    //}
+    public class Cat : Animal
+    {
+        public Cat()
+        {
+
+        }
+
+        public override string Species { get; set; } = "cat";
+
+        public override string Speak()
+        {
+            return "Meow!";
```

Comment:
Watch out for spelling and case.
This is causing a test to fail: `Assert.IsTrue(cat.Speak() == "meow!");`